### PR TITLE
Register sarthak.is-a.dev

### PR DESCRIPTION
**Pull request by dubey0613**
Register sarthak.is-a.dev with CNAME record pointing to https://dubey0613.github.io/portfolio/.

### CODE REVIEW

File: domains/sarthak.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "dubey0613",
+           "email": "dubey.sarthak0613@gmail.com",
+           "discord": "877525618525364295"
+        },
+    
+        "record": {
+            "CNAME": "https://dubey0613.github.io/portfolio/"
```
Comment (CuteDog5695) [inline domains/sarthak.json:9]: Please add more stuff to your website

**Review by iostpa — DISMISSED** [review]

File: domains/sarthak.json (hunk 1 above)
Comment (dubey0613) [reply domains/sarthak.json]: Please Check once again!

File: domains/sarthak.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "dubey0613",
+           "email": "dubey.sarthak0613@gmail.com",
+           "discord": "877525618525364295"
+        },
+    
+        "record": {
+            "CNAME": "https://sarthak-portfolio-web.vercel.app/"
```
Comment (21Z) [inline domains/sarthak.json:9]: `CNAME` can't contain `/` nor protocol (https://)
and we do not support vercel atm

**Review by 21Z — CHANGES_REQUESTED** [review]

File: domains/sarthak.json (hunk 2 above)
Comment (dubey0613) [reply domains/sarthak.json]: Kindly check once again!

File: domains/sarthak.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "dubey0613",
+           "email": "dubey.sarthak0613@gmail.com",
+           "discord": "877525618525364295"
+        },
+    
+        "record": {
+            "CNAME": "dubey0613.github.io/portfolio"
```
Comment (21Z) [inline domains/sarthak.json:9]: `CNAME` can't contain `/` 
```suggestion
            "CNAME": "dubey0613.github.io"
```

**Review by 21Z — CHANGES_REQUESTED** [review]

File: domains/sarthak.json (hunk 3 above)
Comment (dubey0613) [reply domains/sarthak.json]: Do check again!

**Review by 21Z — APPROVED** [review]

**Review by omsenjalia — APPROVED** [review]

**Pull request merged**